dojo/tools/anchore_engine/parser.py

```diff
@@ -30,6 +30,7 @@ def get_findings(self, filename, test):
             findingdetail += '**Feed**: ' + item['feed'] + '/' + item['feed_group'] + '\n\n'
             findingdetail += '**CVE**: ' + cve + '\n\n'
             findingdetail += '**CPE**: ' + item['package_cpe'] + '\n\n'
+            findingdetail += '**Description**: ' + item.get('description', '') + '\n\n'
```

Comment:
As this field is not in the default anchore's json, could we actually fully hide if under an `if` if there is nothing to be seen instead of having an empty description?

Reply:
Changed to not append an empty description.